hudi-common/src/main/java/org/apache/hudi/common/model/HoodieWriteStat.java

```diff
@@ -344,4 +373,18 @@ public void setTotalCreateTime(long totalCreateTime) {
       this.totalCreateTime = totalCreateTime;
     }
   }
+
+  private static Map<String, HoodieColumnRangeMetadata<Comparable>> mergeRecordsStats(
```

Comment:
did we have tests for this against HoodieDeltaWriteStat.
if yes, we should move those tests to operate on HoodieWriteStat

Reply:
then, can we add tests for these.